node/bin/src/main.rs

```diff
@@ -14,6 +16,8 @@ use zksync_os_observability::PrometheusExporterConfig;
 use zksync_os_state::StateHandle;
 use zksync_os_state_full_diffs::FullDiffsState;
 
+const GRACEFUL_SHUTDOWN_TIMEOUT: Duration = Duration::from_secs(10);
```

Comment:
maybe make confiugrable? I can imagine locally we'd want to wait less than on real envs